klee-dev.md

```diff
@@ -1,10 +1,13 @@
 ---
 layout: default
 title: klee-dev
-subtitle: KLEE Developer Mailing List
+subtitle: KLEE Mailing List
 ---
 
-If you have questions about KLEE that are not answered on this website, please send a message to the [klee-dev](https://mailman.ic.ac.uk/mailman/listinfo/klee-dev) mailing list.
-Note that only subscribers can post messages to the list.
+The main list for KLEE discussion, for both users and developers, is [klee-dev](https://mailman.ic.ac.uk/mailman/listinfo/klee-dev) mailing list.
```

Comment:
This sounds weird: "The main list [..] is klee-dev mailing list". Maybe "For both ... the klee-dev mailing list is the central place for discussions, questions, and announcements"